tools/dockerfile/test/bazel/Dockerfile

```diff
@@ -15,58 +15,24 @@
 # Pinned version of the base image is used to avoid regressions caused
 # by rebuilding of this docker image. To see available versions, you can run
 # "gcloud container images list-tags gcr.io/oss-fuzz-base/base-builder"
-# Image(c7f1523ebd92) is built on Jul 29, 2021
-FROM gcr.io/oss-fuzz-base/base-builder@sha256:c7f1523ebd9234b9ff57e5240f8c06569143373be019c92f1e6df18a1e048f37
+# Image(5eceb81f5759) is built on Jan 31, 2022
+FROM gcr.io/oss-fuzz-base/base-builder@sha256:5eceb81f57599d63ca7c9a70c8968b23b128119699626ca749017019eb0b523f
 
 # -------------------------- WARNING --------------------------------------
 # If you are making changes to this file, consider changing
 # https://github.com/google/oss-fuzz/blob/master/projects/grpc/Dockerfile
 # accordingly.
 # -------------------------------------------------------------------------
 
-# Install basic packages and Bazel dependencies.
-RUN apt-get update && apt-get install -y software-properties-common python-software-properties
-RUN add-apt-repository ppa:webupd8team/java
+# Install basic packages
 RUN apt-get update && apt-get -y install \
   autoconf \
   build-essential \
   curl \
-  wget \
   libtool \
   make \
-  openjdk-8-jdk \
-  vim
-
-#====================
-# Python dependencies
-
-# Install dependencies
-# TODO(jtattermusch): This installs python3.5. Is it even needed
-# when we install python3.6 in the next step?
-RUN apt-get update && apt-get install -y \
-    python3-all-dev
-
-#=================
-# Compile CPython 3.6.9 from source
-
-RUN apt-get update && apt-get install -y zlib1g-dev libssl-dev && apt-get clean
-RUN apt-get update && apt-get install -y jq build-essential libffi-dev && apt-get clean
-
-RUN cd /tmp && \
-    wget -q https://www.python.org/ftp/python/3.6.9/Python-3.6.9.tgz && \
-    tar xzvf Python-3.6.9.tgz && \
-    cd Python-3.6.9 && \
-    ./configure && \
-    make -j4 && \
-    make install
-
-RUN cd /tmp && \
-    echo "ff7cdaef4846c89c1ec0d7b709bbd54d Python-3.6.9.tgz" > checksum.md5 && \
-    md5sum -c checksum.md5
-
-RUN python3.6 -m ensurepip && \
-    python3.6 -m pip install coverage
-
+  vim \
+  wget
 
```

Comment:
Removing Python3.6 should be fine for Python Bazel tests. We added 3.6 for Python Bazel tests back then because the default Python was too old, see https://github.com/grpc/grpc/pull/20727 https://github.com/grpc/grpc/issues/20723

Reply:
Thanks for the context! The built-in Python version of new image is 3.8.3 so it appears to be enough.